src/hu_stuff.c

```diff
@@ -1438,8 +1461,7 @@ void WI_DrawTimeWidget(void)
   if (hud_level_time & HUD_WIDGET_HUD)
   {
     HUlib_reset_align_offsets();
-    // leveltime is already added to totalleveltimes before WI_Start()
```

Comment:
The comment was here for a reason. This won't work from the second map in a row, because `totalleveltimes` will be added twice to your total time. 

I understand that we must get rid of the `btuse` part of the widget before drawing it on the intermissing screen. I'll think about a solution...

Reply:
My solution in this PR differs from the one I employed in Nugget, which IIRC was to rebuild the widget in `G_DoCompleted()`, leaving `WI_DrawTimeWidget()` untouched. Would that be better?

Reply:
Yes, we could do it like this and live with the code duplication.

Reply:
That is, we would need a stripped down copy of the total + level time calculation from `HU_widget_build_sttime()` and call it right here.

Reply:
My approach was, specifically, to define a non-static function whose sole purpose is to call `HU_widget_build_sttime()`, and use it in `G_DoCompleted()`. That's pretty much the same as just exposing the build function itself, though, and it doesn't get rid of the `gameaction` check.

Reply:
Alright, let's see that approach!

Reply:
I can get to work on it and the other changes tomorrow. Just a heads-up.

Reply:
Alright, done in 2f35827f9c63d22c5eb42188f8d72c3b18a088cb.